### ISSUE TITLE
Finetune the Autoencoder

### ISSUE THREAD

**Issue by liuzhihui2046** [lllyasviel/ControlNet#292]
Hello, thank you for your work. As the current autoencoder is not very consistent with my existing data, I would like to fine-tune the encoder and decoder based on this. However, I cannot find the corresponding training configuration, especially for the loss configuration. Could you  please provide me with some help? Thank you very much!

**Comment by YihanHu-2022** [comment]
For the finetuning of autoencoder you can refer to the settings of "Latent Diffusion Model" (https://github.com/CompVis/latent-diffusion)

**Comment by GISStd** [comment]
I meet the same question，how do you deal with it
